ocs_ci/ocs/uninstall.py

```diff
@@ -54,6 +55,68 @@ def remove_ocp_registry_from_ocs(platform):
         log.info("platform registry not supported")
 
 
+def uninstall_lso():
+    """
+    Function uninstalls local-volume objects from OCS cluster
+
+    """
+    ocp_obj = ocp.OCP()
+    storage_cluster = ocp.OCP(kind=constants.STORAGECLUSTER,
+                              resource_name=constants.DEFAULT_CLUSTERNAME,
+                              namespace='openshift-storage')
+
+    sc_name = storage_cluster.get().get('spec').get(
+        'storageDeviceSets')[0].get('dataPVCTemplate').get('spec').get('storageClassName')
```

Comment:
```python
sc_name = (
    storage_cluster.get().get('spec').get('storageDeviceSets')[0]
    .get('dataPVCTemplate').get('spec').get('storageClassName')
)
```

Reply:
then moving the . get one line down it adds \, then deleting it, error is marked

Reply:
not fixed